webapp/channels/src/components/admin_console/system_user_detail/system_user_detail.test.tsx

```diff
@@ -133,6 +135,203 @@ describe('SystemUserDetail', () => {
 
         expect(container).toMatchSnapshot();
     });
+
+    describe('change detection', () => {
+        test('should detect email changes and enable save', async () => {
+            const setNavigationBlockedMock = jest.fn();
```

Comment:
Nit: We don't need to make a new mock for this each time. Mocks are reset automatically, so if we remove this, we can just check if `baseProps.setNavigationBlocked` was called. We could probably also change `baseProps.intl.formatMessage` to use the same implementation for all tests as well